packages/playground/website-deployment/README.md

```diff
@@ -0,0 +1,28 @@
+# Hosting playground.wordpress.net on WP Cloud
+
+This document covers our setup to host playground.wordpress.net on WP Cloud.
+
+## Web server configuration
+
+The WP Cloud platform is primarily focused on hosting WordPress. Since playground.wordpress.net is not a WordPress site and has some Playground-specific requirements, we had to do a number of things to get the site running as desired.
+
+WP Cloud uses Nginx which does not support the .htaccess files we used previously for:
+
+-   rewriting URLs
+-   redirecting requests
+-   customizing response headers
```

Comment:
I'm not sure where the multiple spaces between the hyphen and the text came from. Perhaps it is from an auto-formatter on commit.

Reply:
Yeah that’s a formatter. I don’t mind but if you find it annoying then let’s change it.